Workshop2- Homework/mycompany/employees/views.py

```diff
@@ -0,0 +1,53 @@
+from django.views import View
+from .models import Employees
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.utils.decorators import method_decorator
+import json
+
+@method_decorator(csrf_exempt, name='dispatch')
+class EmployeeView(View):
```

Comment:
@Rahul-H-Raj Please add the API to update the employee salary as mentioned in the homework description.

Reply:
Okay I shall see to it

Reply:
@anushds Okay I shall see to it.